### ISSUE TITLE
MusicXML import broken? in 7.1

### ISSUE THREAD

**Issue by lsalgueiro** [openmusic-project/openmusic#45]
Hello all,
Hello dear Karim,

MusicXML import seems to be broken in version 7.1. I have just tried, and it must be something introduced in this version, as 7.0 works.

Here's the Listener output:

```
OM > "ERROR: In = of (1 nil) arguments should be of type number.

Call to om-lisp::om-error-handler
Call to invoke-debugger
Call to error
Call to (method real-group-p (group))
Call to (method get-real-groups (chord))
Call to |MusicXML|::getallgroups
Call to |MusicXML|::groupnotation
Call to (method |MusicXML|::cons-xml-expr (chord))
Call to (method |MusicXML|::cons-xml-expr (group))
Call to (method |MusicXML|::cons-xml-expr (measure))
Call to (method |MusicXML|::cons-xml-expr (voice))
Call to (method |MusicXML|::cons-xml-expr (poly))
Call to (method xml-export (poly))
Call to capi::internal-funcall-using-callback-type
Call to capi::execute-callback
Call to capi::execute-callback-1
Call to capi:execute-with-interface
Call to (subfunction capi-cocoa-library::|%FOREIGN-CALLABLE/CAPI_COCOA_LIBRARY MENU_REPRESENTATION ObjC instance method _CAPIMenuItemSelected:| (fli:define-foreign-callable \"CAPI_COCOA_LIBRARY MENU_REPRESENTATION ObjC instance method _CAPIMenuItemSelected:\"))
Call to objc::|invoke (FUNCTION (OBJC-OBJECT-POINTER SEL OBJC-OBJECT-POINTER OBJC-OBJECT-POINTER OBJC-OBJECT-POINTER) VOID)|
Call to (subfunction 2 objc::make-invoke-function)
Call to objc::invoke-into*
Call to objc:invoke-into
Call to (method capi-library:representation-popup-menu (capi-cocoa-library::screen-representation t t t t t t))
Call to capi:display-popup-menu
Call to (method om-api::om-context-menu-callback (om-graphic-object t t))
Call to capi::in-process-capi-event-callback
Call to capi:execute-with-interface
Call to capi-internals:process-capi-event-with-handler
Call to capi-cocoa-library::handle-mouse-down
Call to (subfunction capi-cocoa-library::|%FOREIGN-CALLABLE/CAPI_COCOA_LIBRARY OUTPUT_PANE_REPRESENTATION ObjC instance method rightMouseDown:| (fli:define-foreign-callable \"CAPI_COCOA_LIBRARY OUTPUT_PANE_REPRESENTATION ObjC instance method rightMouseDown:\"))
Call to objc::|invoke (FUNCTION (OBJC-OBJECT-POINTER SEL) VOID)|
Call to (subfunction 2 objc::make-invoke-function)
Call to objc::invoke-into*
Call to capi-cocoa-library::run-cocoa-event-loop
Call to mp::process-sg-function
Call to mp::run-main-process-once
Call to mp::multiprocessing-main-loop
Call to mp::shared-initialize-multiprocessing
"
```

**Comment by karimhaddad** [comment]
Dear Isalgueiro, 

Thanx for reporting this. Can you send me the patch or POLY that generates this error. I am not sure if this was already fixed or not. 

Best regards
K

**Comment by karimhaddad** [comment]
Hi Luis, 

can you try with this patch?
[fix-7.1-1.lisp.zip](https://github.com/openmusic-project/openmusic/files/10930592/fix-7.1-1.lisp.zip)
